tests/unit/amo/components/TestRatingManager.js

```diff
@@ -605,6 +676,49 @@ describe(__filename, () => {
       expect(root.find(AddonReviewCard)).toHaveLength(0);
       expect(root.find(AddonReviewManager)).toHaveLength(1);
     });
+
+    it('submits a new rating', () => {
+      const { store } = dispatchSignInActions();
+      const addon = createInternalAddon(fakeAddon);
+      const dispatchSpy = sinon.spy(store, 'dispatch');
+      const errorHandler = createStubErrorHandler();
+      const rating = 5;
+      const version = fakeAddon.current_version;
+
+      const root = renderInline({ addon, errorHandler, store, version });
+
+      root.instance().onSelectRating(rating);
```

Comment:
Since you don't need to wait for the promise (hooray), you can do this without reaching into the implementation of `RatingManager`. You can just get it from the `UserRating` configuration:

```diff
diff --git a/tests/unit/amo/components/TestRatingManager.js b/tests/unit/amo/components/TestRatingManager.js
index 6fd03ebfe..c177d6f67 100644
--- a/tests/unit/amo/components/TestRatingManager.js
+++ b/tests/unit/amo/components/TestRatingManager.js
@@ -687,7 +687,9 @@ describe(__filename, () => {
 
       const root = renderInline({ addon, errorHandler, store, version });
 
-      root.instance().onSelectRating(rating);
+      const review = root.find(UserRating);
+      const onSelectRating = review.prop('onSelectRating');
+      onSelectRating(5);
 
       sinon.assert.calledWith(
         dispatchSpy,
```